src/hooks/spaces/useSpaceParams.ts

```diff
@@ -17,8 +20,8 @@ export const useSpaceParams = () => {
   } = useParams<SpaceParams>();
 
   // fallback for ?foo=bar pattern in case it is used as a redirect back
-  const spaceSlugFromQuery = useSearchParam("spaceSlug");
-  const worldSlugFromQuery = useSearchParam("worldSlug");
+  const spaceSlugFromQuery = useSearchParam("spaceSlug") as SpaceSlug;
+  const worldSlugFromQuery = useSearchParam("worldSlug") as WorldSlug;
```

Comment:
One of the rare places !!!assertion!!! shouldn't be called for and even this looks suspicious 😄